tests/int/t1.nim

```diff
@@ -52,3 +52,10 @@ block: # bug #23954
   doAssert testRT_u8 == 7
   const testCT_u8 : uint8 = 0x107.uint8
   doAssert testCT_u8 == 7
+
+block: # issue #24014
```

Comment:
Confused, this looks like my repro from https://github.com/nim-lang/Nim/issues/24104 not https://github.com/nim-lang/Nim/pull/24014

Reply:
Thanks for noticing lol I manually type issue numbers for some reason